docs/getting-started.md

```diff
@@ -59,12 +59,29 @@ scalar.cassy.server.slave_command_path=/path/to/cassy/build/install/cassy/bin
 # URI of a blob store to manage backup files 
 scalar.cassy.server.storage_base_uri=s3://your-bucket
 
+# Enum to tell Cassy which cloud service to use (aws_s3, azure_blob)
+scalar.cassy.server.storage_type=aws_s3
+
 # URL of JDBC for managing some metadata such as backup and restore histories
 scalar.cassy.server.metadata_db_url=jdbc:sqlite:cassy.db
 
 # URL of SRV record. It is used to know and pause Cassandra application nodes to take a cluster-wide snapshot. If you don't use the feature, it can be omitted or the value can be left blank.
 scalar.cassy.server.srv_service_url=_app._tcp.your-service.com
 ```
+### Cloud Storage Type Configuration
+To change the cloud service Cassy uses, the following properties of the configuration file will need to be modified:
+1. `storage_base_uri`: the url to your S3 bucket or Azure blob.
```

Comment:
In my understanding, `storage_base_uri` should be the container name if it is azure blob storage.
Please confirm

Reply:
Could you share your configuration file? I'm sorry, I should have left an example.

During my testing, I set `storage_base_uri = https://cassydev.blob.core.windows.net/indetail-cassy-test`

`indetail-cassy-test` is the name of the blob container.

Reply:
My current working configuration
```
scalar.cassy.server.ssh_user=centos
scalar.cassy.server.ssh_private_key_path=/home/centos/.ssh/id_rsa
scalar.cassy.server.slave_command_path=/opt/cassy/build/install/cassy/bin
scalar.cassy.server.storage_base_uri=scalar-test
scalar.cassy.server.storage_type=AZURE_BLOB
scalar.cassy.server.metadata_db_url=jdbc:sqlite:/home/centos/cassy/cassy.db
scalar.cassy.server.srv_service_url=
```

Reply:
I got a few exceptions when I used `scalar.cassy.server.storage_base_uri=https://cassydev.blob.core.windows.net/scalar-test`

```
Caused by: com.azure.storage.blob.models.BlobStorageException: Status code 400, "?<?xml version="1.0" encoding="utf-8"?><Error><Code>InvalidResourceName</Code><Message>The specifed resource name contains invalid characters.
RequestId:26918a13-601e-0015-7a13-030763000000
Time:2020-03-26T02:12:12.1356498Z</Message></Error>"
```

Reply:
@scalar-boney Sorry, I was wrong. We changed the code so it should only be the name of the container. `scalar.cassy.server.storage_base_uri=scalar-test` was correct. Does it work?

Reply:
>  Does it work?
yes.

Sorry, 
Now I am facing another issue in the backup restore.
Now I have configured Cassandra and cassy in local machine but backup restore is failed.
I have attached  [test details herewith](https://docs.google.com/spreadsheets/d/10gBm2HdhkTibxUYwfhn9v6Qowo4kinZwfH5_h3_OzyQ/edit?usp=sharing)
Could you please check.

Reply:
Sorry,
I have tried cassy with s3 storage but backup restore command failed.
Details attached [here](https://docs.google.com/spreadsheets/d/10gBm2HdhkTibxUYwfhn9v6Qowo4kinZwfH5_h3_OzyQ/edit?usp=sharing)